curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba41a77db20a93f9d7a1.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 66c8ba41a77db20a93f9d7a1
+title: Step 1
+challengeType: 1
+dashedName: step-1
+---
+
+# --description--
+
+In this workshop, you'll learn JavaScript conditionals by building a loan qualification checker app.
+
+The app will check whether the user is eligible for a duplex, car, and condo loan.
+
+The first thing you need to do is to ask the user for the annual income and credit score. For this, you'll have to use the `prompt()` function.
+
+Use `let` to create an `annualIncome` and `creditScore` variables. Set them to `55000` and `650` respectively.
```

Comment:
Any time we have them use `let` because of reassignment, we should mention that.
Because in the past, campers didn't always know why they were writing `let` over `const` and they just felt like they were following directions without knowing why.